doc/user_guide/ecosystem.rst

```diff
@@ -1,20 +1,131 @@
 .. _ecosystem:
 
-Ecosystem
-=========
+Resources & Ecosystem
+=====================
 
-This is a list of projects related to or built on top of Altair. We hope to make it easier to find these projects by listing them here. If you know of a project that should be added, please let us know.
+We hope to make it easier to find these packages and learning resources by listing them here. If you know of a project that should be added, please let us know.
+
+.. _learning-resources:
+
+Learning Resources
+------------------
+
+This is a list of a resources that complements the official documentation and can help you learn more about how to use Altair.
+
+`Visualization Curriculum`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A data visualization curriculum from the UW data group that developed Vega-Lite.
+
+.. List of links.
+.. _`Visualization Curriculum`: https://uwdata.github.io/visualization-curriculum
+
+`Jupyter Notebook Tutorials`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Jupyter Notebooks tutorials and examples from the Altair authors.
+
+.. List of links.
+.. _`Jupyter Notebook Tutorials`: https://github.com/altair-viz/altair_notebooks
+
+`Pycon Tutorial`_
+~~~~~~~~~~~~~~~~~
+
+Altair tutorial given at PyCon 2018 by the Altair authors.
+
+.. List of links.
+.. _`Pycon tutorial`: https://altair-viz.github.io/altair-tutorial
+
+`Data Visualization Course`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This course covers how to create common statistical visualizations, tell stories with data, create geographical visualizations, and bring plots to life by adding interactive elements. Created at the University of British Columbia and can either be audited or taken as part of the `Key Capabilities for Data Science`_ certificate program.
+
+.. List of links.
+.. _`Data Visualization Course`: https://viz-learn.mds.ubc.ca
+.. _`Key Capabilities for Data Science`: https://extendedlearning.ubc.ca/programs/key-capabilities-data-science
+
+`Brief Introduction Videos`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Calmcode provides a few videos that give a brief overview of the Altair library.
+
+.. List of links.
+.. _`Brief Introduction Videos`: https://calmcode.io/altair/introduction.html
+
+
+.. _altair-ecosystem:
+
+Altair Ecosystem
+----------------
+
+This is a list of projects directs related to Altair. There are many other packages that can be used in tandem with Altair, e.g. `dashboard packages which you can read more about in the answers to this StackOverflow question`_.
+
+.. List of links.
+.. _`dashboard packages which you can read more about in the answers to this StackOverflow question`: https://stackoverflow.com/questions/49833866/making-dashboards-using-altair
+
+Vega-Lite_
+~~~~~~~~~~
+
+The higher-level visualization grammar that Altair implements in Python.
+
+.. List of links.
+.. _Vega-Lite: https://vega.github.io/vega-lite
+
+vl-convert_
+~~~~~~~~~~~
+
+
+VegaFusion_
```

Comment:
Somehow my previous code suggestion wasn't applied completely, only the title, so I pasted it again below.
```suggestion
vl-convert_
~~~~~~~~~~~

Python library for converting Altair/Vega-Lite chart specifications into static images (SVG or PNG) or Vega chart specifications without any external dependencies.

.. List of links.
.. _vl-convert: https://github.com/vega/vl-convert

VegaFusion_
```